aspnetcore/fundamentals/startup.md

```diff
@@ -14,82 +14,80 @@ uid: fundamentals/startup
 ---
 # Application Startup
 
->[!WARNING]
-> This page documents version 1.0.0-rc1 and has not yet been updated for version 1.0.0
-
-<a name=application-startup></a>
-
 By [Steve Smith](http://ardalis.com)
 
-ASP.NET Core provides complete control of how individual requests are handled by your application. The `Startup` class is the entry point to the application, setting up configuration and wiring up services the application will use. Developers configure a request pipeline in the `Startup` class that is used to handle all requests made to the application.
+The `Startup` class configures the request pipeline that handles all requests made to the application.
 
 ## The Startup class
 
-In ASP.NET Core, the `Startup` class provides the entry point for an application, and is required for all applications. It's possible to have environment-specific startup classes and methods (see [Working with Multiple Environments](environments.md)), but regardless, one `Startup` class will serve as the entry point for the application. ASP.NET searches the primary assembly for a class named `Startup` (in any namespace). You can specify a different assembly to search using the *Hosting:Application* configuration key. It doesn't matter whether the class is defined as `public`; ASP.NET will still load it if it conforms to the naming convention. If there are multiple `Startup` classes, this will not trigger an exception. ASP.NET will select one based on its namespace (matching the project's root namespace first, otherwise using the class in the alphabetically first namespace).
+All ASP.NET applications require at least one `Startup` class. When an application starts, ASP.NET searches the primary assembly for a class named `Startup` (in any namespace). You can specify a different assembly to search using the *Hosting:Application* configuration key. It doesn't matter whether the class is defined as `public`; ASP.NET will still load it if it conforms to the naming convention. If there are multiple `Startup` classes, ASP.NET selects one based on its namespace (matching the project's root namespace first, otherwise using the class in the alphabetically first namespace).
```

Comment:
>It doesn't matter whether the class is defined as `public`; ASP.NET will still load it if it conforms to the naming convention. 

What naming convention. I'd rewrite as
The class doesn't need to be public.

If there are multiple `Startup` classes, the startup with a matching namespace is selected, otherwise the alphabetical first namespace.